src/iOS/toga_iOS/libs/core_graphics.py

```diff
@@ -1,6 +1,114 @@
 ##########################################################################
 # System/Library/Frameworks/CoreGraphics.framework
 ##########################################################################
+from ctypes import *
+from ctypes import util
+
+from rubicon.objc.types import register_preferred_encoding
+from rubicon.objc import *
+
+
+######################################################################
+core_graphics = cdll.LoadLibrary(util.find_library('CoreGraphics'))
```

Comment:
At column 17: (F405) 'cdll' may be undefined, or defined from star imports: ctypes, rubicon.objc